test/test_utils/__init__.py

```diff
@@ -74,6 +73,13 @@
 PUBLIC_DLC_REGISTRY = "763104351884"
 
 
+def get_python_invoker(ami_id):
+
+    if ami_id in DLAMI_PYTHON_MAPPING:
+        return DLAMI_PYTHON_MAPPING[ami_id]
+    return None
```

Comment:
Why not use `return DLAMI_PYTHON_MAPPING.get(ami_id)`?

Reply:
Also, please return something like `python3` as default, because using this function on an undefined AMI or a non-DLAMI will cause it to return `None`, and cause tests to fail.

Reply:
The get call can be used as well, will make the change